spec/abstract/KHeap_A.thy

```diff
@@ -233,6 +257,14 @@ where
      return $ f sc
    od"
 
+definition (* update only the schedcontext in place, keeping the size *)
```

Comment:
Is this function not exactly equal to `update_sched_context`?

Reply:
It is not. But this is still not final and not needed for this PR, so I will remove.

Reply:
Hmmm. 

```
lemma asdf:
  "read_update_sched_context ptr f = update_sched_context ptr f"
  apply (clarsimp simp: read_update_sched_context_def update_sched_context_def get_object_def)
  done
  ```

works for me. Or is there some other reason we'd like both at some stage? I guess we can come back to it later

Reply:
Ah I thought you meant literally. This is just an experiment that I forgot to come back and is not meant for this PR. So, as I said, it will be removed.